src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -2404,8 +2405,29 @@ namespace InternalHeatGains {
                             ErrorsFound = true;
                         }
                         thisZoneOthEq.otherEquipFuelTypeString = FuelTypeString; // Save for output variable setup later
-                        state.dataHeatBal->Zone(zoneNum).otherEquipFuelTypes.emplace_back(FuelTypeString);
-                        state.dataHeatBal->space(spaceNum).otherEquipFuelTypes.emplace_back(FuelTypeString);
+                        // Build list of fuel types used in each zone and space (excluding None and Water)
+                        bool found = false;
+                        for (auto fuelType : state.dataHeatBal->Zone(zoneNum).otherEquipFuelTypeNums) {
+                            if (thisZoneOthEq.OtherEquipFuelType == fuelType) {
+                                found = true;
+                                break;
+                            }
+                        }
+                        if (!found) {
+                            state.dataHeatBal->Zone(zoneNum).otherEquipFuelTypeNums.emplace_back(thisZoneOthEq.OtherEquipFuelType);
+                            state.dataHeatBal->Zone(zoneNum).otherEquipFuelTypeNames.emplace_back(FuelTypeString);
```

Comment:
Surprised this had to happen. These weren't saved before?

Reply:
I was trying to keep the list so all fuel types would be reported a the zone and space level (for other equipment) but ran out of time. The usage is only tracked for one at the zone/space level. But I left this here so the warning messages could list everything that's not available. It's all metered appropriately and reportable at the object level, just a single fuel type at the space/zone level.